scripts/jenkins/qa_openstack.sh

```diff
@@ -152,7 +152,7 @@ case "$cloudsource" in
         osrelease=${osrelease^}
         $zypper ar -G -f $cloudopenstackmirror/$osrelease/$REPO/ cloud
         if test -n "$OSHEAD" ; then
-            if [ "Rocky" = "$osrelease" ]; then
+            if [[ "$osrelease" =~ ^[rs] ]]; then
```

Comment:
```suggestion
            if [[ "$osrelease" =~ (Rocky|Stein) ]]; then
```
This is case sensitive.